specs/merge/fork.md

```diff
@@ -120,7 +120,7 @@ def compute_terminal_total_difficulty(anchor_pow_block: PowBlock) -> uint256:
 
 def get_transition_store(anchor_pow_block: PowBlock) -> TransitionStore:
     terminal_total_difficulty = compute_terminal_total_difficulty(anchor_pow_block)
-    return TransitionStore(terminal_total_difficulty=terminal_total_difficulty)
+    return TransitionStore(terminal_total_difficulty=terminal_total_difficulty, terminal_block_hash=None)
```

Comment:
lint error:
```
eth2spec/merge/mainnet.py:2762: error: Argument "terminal_block_hash" to "TransitionStore" has incompatible type "None"; expected "Hash32"
```

You have to change the `terminal_block_hash` type to:
```python
@dataclass
class TransitionStore(object):
    terminal_total_difficulty: uint256
    terminal_block_hash: Optional[Hash32]
```

Reply:
As mentioned above:
```suggestion
    return TransitionStore(terminal_total_difficulty=terminal_total_difficulty, terminal_block_hash=Hash32())
```

Reply:
Prefer the default to `Hash32`. Makes the spec read cleaner 👍